native/golang/utils/bytes.go

```diff
@@ -44,3 +46,24 @@ func CollectContentsAsBytes(input string, isByteString bool) (data []byte, err e
 
 	return
 }
+
+func LoadChecksums() map[string]string {
+	checksumsFile := "checksums.asc"
+	file, err := os.Open(checksumsFile)
+	if os.IsNotExist(err) {
+		return nil
+	} else if err != nil {
+		panic(err)
```

Comment:
Can we add a `utils.LogError` here to track weird edge-cases, specifically there is an error that isn't related to the absence of a checksums file?